src/main/java/io/opensergo/OpenSergoClient.java

```diff
@@ -56,17 +59,68 @@ public OpenSergoClient(String host, int port) {
         this.configCache = new SubscribedConfigCache();
         this.subscribeRegistry = new SubscribeRegistry();
         this.reqId = new AtomicInteger(0);
+        status = OpensergoClientStatus.INITIAL;
+    }
+
+    public void registerSubscribeInfo(OpensergoClientSubscribeInfo subscribeInfo) {
+        // Register subscriber to local.
+        if (Optional.of(subscribeInfo.getSubscriberList()).isPresent() && subscribeInfo.getSubscriberList().size() > 0) {
+            subscribeInfo.getSubscriberList().forEach(subscriber -> {
+                this.subscribeRegistry.registerSubscriber(subscribeInfo.getSubscribeKey(), subscriber);
+                OpenSergoLogger.info("OpenSergo subscribeinfo registered, subscribeKey={}, subscriber={}", subscribeInfo.getSubscribeKey(), subscriber);
+
+                if (requestAndResponseWriter != null && this.status == OpensergoClientStatus.STARTED) {
+                    this.subscribeConfig(subscribeInfo.getSubscribeKey());
+                }
+            });
+        }
     }
 
     public void start() throws Exception {
+        OpenSergoLogger.info("OpensergoClient is starting...");
+
+        if (status == OpensergoClientStatus.INITIAL) {
+            OpenSergoLogger.info("open keepavlive thread");
+            new Thread(this::keepAlive).start();
```

Comment:
This thread SHOULD be named and daemon. And be careful with the task management.

Reply:
Actually, I was confused on how to judge the num of threads. Because we didn't know how many OpenSergoClient will be created by user, so the number for core-threads in ThreadPool is a question.

Reply:
finished.